doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -3053,36 +3053,50 @@ The following table indicates the software maturity levels of the support for ea
               - nRF54L05
               - nRF54L10
               - nRF54L15
+              - nRF54LM20
```

Comment:
It was communicated that the these devices should not be mentioned anywhere in the documentation until the preview-release in September.  The only exception is the sample_board_rows file, to enable running tests and generating tables from yaml.